docs/index.md

```diff
@@ -37,3 +37,14 @@ For more details on the evolution of OLM and the roadmap for v1, explore the fol
 
 - [Multi-Tenancy Challenges, Lessons Learned, and Design Shifts](project/olmv1_design_decisions.md)
 - [OLM v1 Roadmap](project/olmv1_roadmap.md)
+
+## Can I Migrate from OLMv0 to OLMv1?
+
+Not yet. Currently, OLMv1 is a **Minimum Viable Product (MVP)** with a subset of OLMv0’s functionality.
```

Comment:
Here are some suggested text. 
```
yes, but with [limitations](project/olmv1_limitations.md). OLM v1 is actively working to address some of the limitations. However because OLMv1's design principles are different than OLMv0 it is hard for us to say if they will be fully compatible with each-other in future.  We will update this space with more information when they are available.

Reply:
I do not think we need to provide more information here. IMO less is more here.